### PR TITLE
fix(elixir): use local Rust `extism` package from `extism_nif` for local development

### PR DESCRIPTION
**Pull request by zshipko**

### CODE REVIEW

File: elixir/native/extism_nif/Cargo.toml [hunk 1]
```diff
@@ -11,5 +11,5 @@ crate-type = ["cdylib"]
 
 [dependencies]
 rustler = "0.26.0"
-extism = { version = "0.1.0" }
+extism = { version = "0.1.0", path = "../../../rust" }
```
Comment (bhelx) [inline elixir/native/extism_nif/Cargo.toml:14]: I had to remove this for a reason. Can't quite remember why. I think it was failing on release. I'll test it out tomorrow.

Comment (nilslice) [reply elixir/native/extism_nif/Cargo.toml:14]: @bhelx - did you get a chance to test this out? just don't want to let it sit too long

Comment (bhelx) [reply elixir/native/extism_nif/Cargo.toml:14]: if i remember correctly, it was breaking on release. let me see if there is a way i can dry run it

Comment (bhelx) [reply elixir/native/extism_nif/Cargo.toml:14]: `mix hex.publish --dry-run` worked

**Pull request merged**